ui/components/app/snaps/keyring-snap-removal-warning/keyring-snap-removal-warning.tsx

```diff
@@ -152,6 +152,10 @@ export default function KeyringRemovalSnapWarning({
                       validateConfirmationInput(e.target.value),
                     );
                   }}
+                  onPaste={(e: React.ClipboardEvent<HTMLInputElement>) => {
+                    e.preventDefault();
+                    return false;
```

Comment:
I don't think we need to return anything here? Aren't all event handlers void functions?

Reply:
Yea, you're right, this can be removed